lib/questions.rb

```diff
@@ -97,12 +116,14 @@ def average_of_array(array)
 # [1, 3, 5, 4, 1, 2, 6, 2, 1, 3, 7]
 # becomes [1, 3, 5, 4, 1, 2]
 def get_elements_until_greater_than_five(array)
+  array.first(array.find_index{|x|x>5})
```

Comment:
Space after closing `|` missing.<br>Surrounding space missing for operator '>'.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.